suites/pacific/rgw/tier1_rgw_multisite_primary_to_secondary.yaml

```diff
@@ -520,3 +520,15 @@ tests:
             config-file-name: test_bucket_policy_replace.yaml
             verify-io-on-site: ["ceph-sec"]
             timeout: 300
+
+  - test:
+      name: Bilog trimming test
+      desc: test bilog trimming
+      polarion-id: CEPH-83572658
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_bilog_trimming.py
+            config-file-name: test_bilog_trimming.yaml
+            timeout: 300
```

Comment:
Could you move this test to `tier_1_rgw_extn...`? IMO, let's avoid adding more tests to stage 1 run